images/clickhousenozk/config/supervisor/conf.d/clickhouse.conf

```diff
@@ -2,7 +2,7 @@
 command=/usr/bin/clickhouse-server --config /etc/clickhouse-server/config.xml
 process_name=%(program_name)s
 autostart=true
-autorestart=true
+autorestart=false
```

Comment:
Maybe "unexpected" with exitcodes (0 by default) is more suitable ? To cover failing terminations.